lib/crt.php

```diff
@@ -0,0 +1,32 @@
+<?php
+require_once('preload.php');
+
+final class CRT {
+    private static $ffi = null;
+    function __construct() {
+        if (is_null(self::$ffi)) {
+            self::$ffi = FFI::scope("AWSCRT");
+        }
+    }
+
+    function init() {
+        return self::$ffi->aws_crt_init();
+    }
+
+    function clean_up() {
+        return self::$ffi->aws_crt_clean_up();
+    }
+
+    function last_error() {
+        return self::$ffi->aws_crt_last_error();
+    }
+
+    function error_str($error) {
+        return self::$ffi->aws_crt_error_str((int)$error);
```

Comment:
Should we add a single space between cast operator and variable? Not sure what is current convention.

Reply:
Sure. Will do.

Reply:
Better yet, is there a decent linter/formatter we could use?